pkgs/by-name/ru/ruff-lsp/package.nix

```diff
@@ -50,7 +42,7 @@ python3Packages.buildPythonApplication rec {
 
   makeWrapperArgs = [
     # prefer ruff from user's PATH, that's usually desired behavior
-    "--suffix PATH : ${lib.makeBinPath [ ruff ]}"
+    "--suffix PATH : ${lib.makeBinPath (with python3Packages; [ ruff ])}"
```

Comment:
I think the wrapper can be removed in this setup, at least in the past ruff-lsp had code that were searching for ruff in adjacent ruff python package, the wrapper was needed because we didn't have this package installed.

Reply:
Would you mind taking over this PR?
I don't use ruff-lsp myself, and I can't test changes.

Reply:
Sorry I don't have time at the moment and I don't use it either anymore, I was hopping astral will make it deprecated soon so we could remove it from nixpkgs.

Reply:
So this can't be removed because ruff-lsp doesn't use the python library to lookup the path to ruff.